src/sql/ConfigEditor/ConfigSelect.tsx

```diff
@@ -3,17 +3,17 @@ import { DataSourcePluginOptionsEditorProps, SelectableValue } from '@grafana/da
 import { InputActionMeta } from '@grafana/ui';
 import { AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData } from '../../types';
 import { ResourceSelector } from '../ResourceSelector';
-import { DEFAULT_LABEL_WIDTH } from '../../ConnectionConfig';
 
 export interface ConfigSelectProps
   extends DataSourcePluginOptionsEditorProps<AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData> {
   value: string;
+  // input id and aria-label necessary for accessibility attributes
+  id: string;
+  ['aria-label']: string;
```

Comment:
Why is `aria-label` in brackets?

Reply:
It's a way to define html element props that aren't camel case, like aria-label and data-testid so that typescript linter doesn't complain. I could have made the ariaLabel an 'official' component prop as well, of course. However, I replaced it in the final version with just `label` props.